front_matter/any.ts

```diff
@@ -28,9 +34,11 @@ export type { Extractor };
  * result.body; // "Hello, world!"
  * result.attrs; // { title: "Three dashes marks the spot" }
  * ```
+ *
+ * @typeParam T The type of the parsed front matter.
+ * @param text The text to extract front matter from.
+ * @returns The extracted front matter and body content.
  */
-export const extract: Extractor = createExtractor({
-  yaml: parseYaml as Parser,
-  toml: parseToml as Parser,
-  json: JSON.parse as Parser,
-});
+export function extract<T>(text: string): Extract<T> {
+  return _extractor(text);
```

Comment:
```suggestion
  return createExtractor({
    yaml: parseYaml as Parser,
    toml: parseToml as Parser,
    json: JSON.parse as Parser,
  })(text);
```

Reply:
`createExtractor` is relatively expensive. I don't think it's good idea to call it each time.

Reply:
Good point. Never mind.